commit 57bd4a75e21a2723f64af2675e001b6f61c91474 (HEAD -> master)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-09-13T10:57:19-07:00 (3 weeks ago)
    
    TOOLS-2254 add redirects from mo-docs to modocs.joyent.us (restdown files should now always be .md)
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/Makefile b/Makefile
index 13b7f9e..3e0d87b 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ RESTDOWN_FLAGS  := -b deps/restdown/brand/spartan
 #
 # Files
 #
-DOC_FILES	 = index.restdown motivation.restdown workflowapi.restdown
+DOC_FILES	 = index.md motivation.md workflowapi.md
 JS_FILES	:= $(shell ls *.js) $(shell find lib test -name '*.js')
 JSL_CONF_NODE	 = tools/jsl.node.conf
 JSL_FILES_NODE   = $(JS_FILES)
diff --git a/docs/index.restdown b/docs/index.md
similarity index 100%
rename from docs/index.restdown
rename to docs/index.md
diff --git a/docs/motivation.restdown b/docs/motivation.md
similarity index 100%
rename from docs/motivation.restdown
rename to docs/motivation.md
diff --git a/docs/workflowapi.restdown b/docs/workflowapi.md
similarity index 100%
rename from docs/workflowapi.restdown
rename to docs/workflowapi.md
